vmicore/src/io/ILogger.h

```diff
@@ -4,63 +4,63 @@
 #include "cxxbridge/rust/cxx.h"
 #include "cxxbridge/rust_grpc_server/src/bridge.rs.h"
 #include <memory>
+#include <string_view>
 
 namespace VmiCore
 {
-#define NEW_LOGGER(log_server) ((log_server)->newNamedLogger(std::filesystem::path(__FILE__).filename().stem()))
     // Has to be the same as in the HiveOperations project to indicate in which file the log should be written.
```

Comment:
I think we should try and get rid of these processing internals.

Reply:
Agreed, but not in this PR though.